compiler/rustc_hir/src/def.rs

```diff
@@ -389,7 +396,7 @@ pub enum Res<Id = hir::HirId> {
     ///
     /// **Belongs to the value namespace.**
     ///
-    /// *See also [`Res::SelfTy`].*
+    /// *See also [`Res::SelfTyTrait`] and [`Res::SelfTyTrait`].*
     SelfCtor(DefId),
```

Comment:
Could you put this variant right below `SelfTyOther`/`SelfTyAlias`?

Reply:
Done.